iset.mm

```diff
@@ -125615,6 +125689,168 @@ reduced fraction representation (no common factors, denominator
+      c0 fiintim anim2i syl ) BDEZAFZBGZUEHBEIAJZUECFKBECBLABLZMZUGUFUETNUEOEPU
+      EQBEIAJZMUDUIACDBRSUHUJUGACBUAUBUC $.
+  $}
+
+  ${
+    $d x y A $.  $d y B $.  $d x y J $.
+    $( The union of a subset of a topology (that is, the union of any family of
+       open sets of a topology) is an open set.  (Contributed by Stefan Allan,
+       27-Feb-2006.) $)
+    uniopn $p |- ( ( J e. Top /\ A C_ J ) -> U. A e. J ) $=
+      ( vx vy ctop wcel wss cuni cv wi wal cin wral wa istopg ibi simpld elpw2g
+      cpw biimpar wceq sseq1 unieq eleq1d imbi12d spcgv syl ex pm2.43d mpid imp
+      com23 ) BEFZABGZAHZBFZUMUNCIZBGZUQHZBFZJZCKZUPUMVBUQDILBFDBMCBMZUMVBVCNCD
+      EBOPQUMUNVBUPJZUMUNUNVDJUMUNNZVBUNUPVEABSZFZVBUNUPJZJUMVGUNABERTVAVHCAVFU
+      QAUAZURUNUTUPUQABUBVIUSUOBUQAUCUDUEUFUGULUHUIUJUK $.
+
+    $( The indexed union of a subset of a topology is an open set.
+       (Contributed by NM, 5-Oct-2006.) $)
+    iunopn $p |- ( ( J e. Top /\ A. x e. A B e. J ) -> U_ x e. A B e. J ) $=
+      ( vy ctop wcel wral wa ciun cv wceq wrex cab dfiun2g adantl wss uniiunlem
+      cuni ibi uniopn sylan2 eqeltrd ) DFGZCDGABHZIABCJZEKCLABMENZSZDUEUFUHLUDA
+      EBCDOPUEUDUGDQZUHDGUEUIAEBCDDRTUGDUAUBUC $.
+
+    $( The intersection of two open sets of a topology is an open set.
+       (Contributed by NM, 17-Jul-2006.) $)
+    inopn $p |- ( ( J e. Top /\ A e. J /\ B e. J ) -> ( A i^i B ) e. J ) $=
+      ( vx vy ctop wcel cin cv wral wa wss cuni wi wal istopg ibi simprd eleq1d
+      wceq ineq1 ineq2 rspc2v syl5com 3impib ) CFGZACGZBCGZABHZCGZUFDIZEIZHZCGZ
+      ECJDCJZUGUHKUJUFUKCLUKMCGNDOZUOUFUPUOKDEFCPQRUNUJAULHZCGDEABCCUKATUMUQCUK
+      AULUASULBTUQUICULBAUBSUCUDUE $.
+
+    $( The intersection of a nonempty finite family of open sets is open.
+       (Contributed by FL, 20-Apr-2012.) $)
+    fiinopn $p |- ( J e. Top -> ( ( A C_ J /\ A =/= (/) /\ A e. Fin )
+        -> |^| A e. J ) ) $=
+      ( vx wss c0 wne cfn wcel w3a ctop cint wi elpwg cv wceq sseq1 neeq1 com12
+      cpw wal eleq1 3anbi123d inteq eleq1d imbi2d imbi12d wa sp adantl istopfin
+      cuni syl11 vtoclg 3exp com3r com4r syl6bir pm2.43a com4l pm2.43i 3imp ) A
+      BDZAEFZAGHZIZBJHZAKZBHZVBVCVDVFVHLZVBVCVDVILLVDVBVBVCVIVBVDVBVCVILLZVDVBA
+      BSZHZVDVJLABGMVDVBVCVLVIVBVCVDVLVILZVBVCVDVMVLVEVICNZBDZVNEFZVNGHZIZVFVNK
+      ZBHZLZLVEVILCAVKVNAOZVRVEWAVIWBVOVBVPVCVQVDVNABPVNAEQVNAGUAUBWBVTVHVFWBVS
+      VGBVNAUCUDUEUFVOVNUKBHLCTZVRVTLZCTZUGVRVTVFWEWDWCWDCUHUICBUJULUMRUNUOUPUQ
+      URUSUTVAR $.
+  $}
+
+  $( The union of two open sets is open.  (Contributed by Jeff Madsen,
+     2-Sep-2009.) $)
+  unopn $p |- ( ( J e. Top /\ A e. J /\ B e. J ) -> ( A u. B ) e. J ) $=
+    ( ctop wcel w3a cpr cuni cun wceq uniprg 3adant1 wa wss prssi uniopn sylan2
+    3impb eqeltrrd ) CDEZACEZBCEZFABGZHZABIZCUAUBUDUEJTABCCKLTUAUBUDCEZUAUBMTUC
+    CNUFABCOUCCPQRS $.
+
+  $( The empty set is an open subset of a topology.  (Contributed by Stefan
```

Comment:
of **any** topology

It might be worth also proving the variable-free form `|- I^I Top = { (/) } `.

Reply:
> of **any** topology

I guess that is a bit better, although "a topology" isn't wrong. Changed in set.mm and iset.mm.

> It might be worth also proving the variable-free form `|- |^| Top = { (/) } `.

If you do decide to prove this I'd start out by putting it in your mathbox. At least to my eyes it is harder to read and I'm not sure where we'd use it.

Reply:
"a topology" in the sentence could be understood as "some topology" (at least when I translate it to French, maybe that's not the case in English), which is why "any" is clearer.

The statement I propose also says that the empty set is the only class with that property. I may prove it in set.mm if I find time.